src/context/index.tsx

```diff
@@ -18,7 +18,7 @@ const AppWrapperContext: FC<Props> = ({ children }) => {
     const { edit: editQuery } = query;
     const isEditMode = editQuery === 'true';
     const onEditRoute = () => {
-        router.push({ query: 'edit=true' }, undefined, { shallow: true }); 
+        router.push({ query: 'edit=true' }, undefined, { shallow: true });
```

Comment:
Check the below snippet, and reference url for more info.

```ts
router.push({ query: { edit: 'true' } }, undefined, { shallow: true });

```

Ref: https://nextjs.org/docs/api-reference/next/router#with-url-object